activities/GetThingsDone.activity/js/activity.js

```diff
@@ -1,4 +1,4 @@
-define(["sugar-web/activity/activity","sugar-web/datastore","activity/model","activity/view","activity/controller"], function (activity,datastore,model,view,controller) {
+define(["sugar-web/activity/activity", "sugar-web/datastore", "activity/model", "activity/view", "activity/controller"], "l10n", function (activity, webL10n, datastore, model, view, controller) {
```

Comment:
BTW The module name is `webL10n`, not `L10n`. Additionally, you haven't put into the array.